Jason/Jason-WorkShop-3/app/src/main/res/layout/activity_check_bmi.xml

```diff
@@ -18,6 +18,7 @@
             android:text="HELLO"
             android:textStyle="bold"
             android:textColor="#445460"
```

Comment:
Because with the phone, which using API < 15, they can't use type color of color.xml